src/fonda/step.cljs

```diff
@@ -37,7 +57,9 @@
   (let [step
```

Comment:
What about tap and injected steps? They should also support `:is-anomaly-error?` to be consistent.